src/components/form-elements/text-input/TextInput.tsx

```diff
@@ -8,25 +8,49 @@ interface TextInputProps extends HTMLProps<HTMLInputElement>, FormElementProps {
   inputRef?: MutableRefObject<HTMLInputElement | null>;
   width?: InputWidth;
   disableErrorLine?: boolean;
+  prefix?: string;
+  suffix?: string;
 }
 
+const TextInputPrefix: React.FC<{ prefix: string }> = ({ prefix }) => (
+  <div className="nhsuk-input__prefix" aria-hidden="true">
+    {prefix}
+  </div>
+);
+
+const TextInputSuffix: React.FC<{ suffix: string }> = ({ suffix }) => (
+  <div className="nhsuk-input__suffix" aria-hidden="true">
+    {suffix}
+  </div>
+);
+
 const TextInput: React.FC<TextInputProps> = (props) => (
   <FormGroup<TextInputProps> {...props} inputType="input">
-    {({
-      width, className, error, inputRef, type = 'text', ...rest
-    }) => (
-      <input
-        className={classNames(
-          'nhsuk-input',
-          { [`nhsuk-input--width-${width}`]: width },
-          { 'nhsuk-input--error': error },
-          className,
-        )}
-        ref={inputRef}
-        type={type}
-        {...rest}
-      />
-    )}
+    {({ width, className, error, inputRef, type = 'text', prefix, suffix, ...rest }) => {
+      const Input = (
+        <input
+          className={classNames(
+            'nhsuk-input',
+            { [`nhsuk-input--width-${width}`]: width },
+            { 'nhsuk-input--error': error },
+            className,
+          )}
+          ref={inputRef}
+          type={type}
+          {...rest}
+        />
+      );
+
+      return prefix || suffix ? (
+        <div className="nhsuk-input__wrapper">
+          {prefix ? <TextInputPrefix prefix={prefix} /> : null}
+          {Input}
+          {suffix ? <TextInputSuffix suffix={suffix} /> : null}
+        </div>
+      ) : (
+        Input
+      );
+    }}
```

Comment:
once enzyme migration is in the feature branch this will need unit tests adding

Reply:
Have added note for this to the subtask https://nhsd-jira.digital.nhs.uk/browse/NUT-4646